src/writeData/subscriptions/components/BrokerFormContent.tsx

```diff
@@ -291,6 +297,8 @@ const BrokerFormContent: FC<Props> = ({
                   {feature: 'subscriptions'}
                 )
                 setSecurity('none')
+                formContent.brokerUsername = undefined
+                formContent.brokerPassword = undefined
```

Comment:
can we change this to `null`? I know we're handling both on the backend but I think it would be nice for this to have this set the values correctly in case that `if block` in nifid is ever removed